Sources/ProjectDescription/Target.swift

```diff
@@ -17,8 +17,8 @@ public struct Target: Codable, Equatable {
     /// The product bundle identifier.
     public let bundleId: String
 
-    /// The minimum deployment target your product will support.
-    public let deploymentTarget: DeploymentTarget?
```

Comment:
Note this is a breaking change and will require a major release

suggestion:

- Add an alternate initializer which can proxy the old API to the new one to maintain compatibility with both

Reply:
I agree with this approach

Reply:
I am not sure if we can remove platform. For **frameworks** it really doesn't matter but platform for **apps** is needed because it defines host application for tests so we can choose which platform we are going to use for running tests.

Reply:
@kwridan I added multi_platforms_app to /projects/tuist/fixtures/ to follow the changes